snippets/language-cloudslang.cson

```diff
@@ -13,14 +13,17 @@
     'prefix': 'flow'
     'body': """
       ####################################################
-      # ${17:Flow_description}
-      #
-      # Inputs:
-      #   - ${5:input_name} - ${18:input_description}
-      # Outputs:
-      #   - ${14:output_name} - ${19:output_description}
-      # Results:
-      #   - ${16:result_name} - ${20:result_description}
+      ## description: ${17:Flow_description}
+      ##
+      ## inputs: |
+      ##   - ${5:input_name}: |
+      ##      ${18:input_description}
+      ##      ${18:input_description}
+      ##   - ${5:input_name}: ${13:input_description}
```

Comment:
I think you should either leave only one input in the desc or add another one also for the flow (different number)